app/src/components/Common/DateTimeInput/DateTimeInput.tsx

```diff
@@ -2,29 +2,32 @@ import React from 'react';
 import { DateInput } from '../DateInput/DateInput';
 import { TimeInput } from '../TimeInput/TimeInput';
 import { EditDateTime } from 'src/types/date-time';
+import style from './DateTimeInput.module.scss';
 
-interface Props {
+interface IProps {
   label: string;
   value: EditDateTime;
   onChange: (datetime: EditDateTime) => void;
 }
 
-export const DateTimeInput = ({ label, value, onChange }: Props) => {
+export const DateTimeInput = ({ label, value, onChange }: IProps) => {
   return (
-    <>
+    <section>
       {label}
-      <DateInput
-        label={''}
-        value={value.date}
-        onChange={date => {
-          onChange({ date, time: value.time });
-        }}
-      />
-      <TimeInput
-        label={''}
-        value={value.time}
-        onChange={time => onChange({ date: value.date, time })}
-      />
-    </>
+      <section className={style.grid}>
```

Comment:
Synes ikke den innerste bør være en `section`, bruk like gjerne `div` for den, wrap `label`'en i en `h6` eller noe sånt